CM_dependencies.cmake

```diff
@@ -1,18 +1,23 @@
 # Darknet object detection framework
 
-
 # ==========
 # == CUDA ==
 # ==========
-CHECK_LANGUAGE (CUDA)
```

Comment:
Can you re-do this pull request so it only contains the changed files?

Reply:
I wonder if it is because you deleted line #3 that it somehow thinks every line has changed?

Reply:
ok, and yes, I'm aware it goes against NVidia "recommend" way of doing things but it doesn't create new environment variables it doesn't create new paths and it solves the problems. I have never followed that way it just putting files in a different path. but will I fix it. so, it shows only the changes.

Reply:
The other thing is we need to make sure it doesn't break things on Linux as well.